uk.jnthn.backgammony.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "uk.jnthn.backgammony",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "backgammony",
+    "modules": [
+        {
+            "name": "gnubg",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure",
+                "make datadir=/app/share",
```

Comment:
Pass `-j$FLATPAK_BUILDER_N_JOBS` to make the build parallel, if possible.

Reply:
Why even use `buildsystem: simple` here? It looks like generic autotools.